### PR TITLE
Hacktoberfest 2022 - Adobe Photoshop Q95

### PR DESCRIPTION
**Pull request by nightshadetvn**
Hacktoberfest 2022 - added answer Q95 to photoshop quiz

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1648,6 +1648,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
       <td align="center"><a href="https://github.com/MarkFantini"><img src="https://avatars.githubusercontent.com/u/88554151?v=4?s=100" width="100px;" alt="Marcelo Antunes Soares Fantini"/><br /><sub><b>Marcelo Antunes Soares Fantini</b></sub></a><br /><a href="#design-MarkFantini" title="Design">🎨</a> <a href="https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/commits?author=MarkFantini" title="Code">💻</a></td>
+      <td align="center"><a href="https://github.com/nightshadetvn"><img src="https://avatars.githubusercontent.com/u/46790388?v=4?s=100" width="100px;" alt="nightshadetvn"/><br/><sub><b>nightshadetvn</b></sub></a><br /$<a href="#design-nightshadetvn" title="Design">🎨</a> <a href="https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/commits?author=nightshadetvn" title="Code">💻</a></td>
```
Comment (Ebazhanov) [inline README.md:1651]: could you please revert it, because it is added automatically by all-contribution command

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
thanks but left one comment

**Comment by nightshadetvn** [comment]
Hey @Ebazhanov I fixed the issue ;)

**Review by Ebazhanov — APPROVED** [review]
LGTM, thank you @nightshadetvn for your contribution! 👍

**Comment by Ebazhanov** [comment]
@all-contributors please add @nightshadetvn for design, code

**Pull request merged**